decentraland/unity-explorer
# fix: exclude sidebar camera reel button if cli arg is missing

## What does this PR change?



Removes the sidebar button to open the camera reel ExplorePanel section if the CLI arg to enable the feature is missing.

(Keep in mind that the flag that is passed down to the plugins/controllers is temporary as in all other plugins/controllers and will be removed once the feature is fully released)

## How to test the changes?



1. Launch the explorer with the CLI arg `camera-reel`
2. Check whether the feature is completely enabled (explore panel section, sidebar button, ...)
3. Re-launch the explorer without the CLI arg
4. Check that the feature is completely disabled, especially that the sidebar button is not visible

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
